pennylane/ops/op_math/controlled_class.py

```diff
@@ -140,10 +140,15 @@ def __init__(
         self.hyperparameters["control_values"] = control_values
         self.hyperparameters["work_wires"] = work_wires
 
-        self._name = f"C{base.name}"
+        self._name = f"C:{base.name}"
```

Comment:
How is this change of nomenclature motivated? Will it be consistent throughout the PL codebase (Thinking of `ControlledQubitUnitary`, for example)?

Reply:
I was testing with a broadcasted Controlled RX.  Since the `RX` was broadcasted, the matrix would be undefined and `has_martix` would be `False`.

Unfortunately, the name would then be the same as `CRX`, which is explicitly supported. I needed to modify the name so that it wouldn't be the same as `CRX`.

Reply:
Ohhh, good catch! I suppose it's also nice for transparency to have different names for them, I was not aware of this duplication in the namespace :)